dotnet/samples/KernelSyntaxExamples/Example62_CustomAIServiceSelector.cs

```diff
@@ -81,83 +80,33 @@ public static async Task RunSemanticFunctionAsync(IKernel kernel, string prompt,
     }
 }
 
-public class MyAIServiceSelector : IAIServiceSelector
+public class ByModelIdAIServiceSelector : IAIServiceSelector
```

Comment:
Small typo
```suggestion
public class MyModelIdAIServiceSelector : IAIServiceSelector
```

Reply:
That's the intended name because the code selects "By Model Id"